src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -860,6 +872,19 @@ Copyright (c) .NET Foundation. All rights reserved.
         ItemName="_RestoreGraphEntry" />
     </GetRestorePackageReferencesTask>
 
+    <!-- Write out central package versions -->
+    <GetCentralPackageVersionsTask
+      Condition=" '$(PackageReferenceCompatibleProjectStyle)' == 'true' AND '$(_CentralPackageVersionsEnabled)' == 'true' "
+      ProjectUniqueName="$(MSBuildProjectFullPath)"
+      CentralPackageVersions="@(PackageVersion)"
+      TargetFrameworks="$(TargetFramework)"
+      >
```

Comment:
`>` should be at the end of the line above.

Reply:
The pattern is the same as the other task definitions.